bin/resty

```diff
@@ -894,6 +906,10 @@ Options:
                         in the future).
 
     -V                  Print version numbers and nginx configurations.
+
+    --dump-nginx-conf   Print out the generated nginx configuration file only
+                        without running the Lua code.
```

Comment:
I think we can omit "out" and "only" from this sentence, and maybe be more explicit even:
```
Print the generated nginx configuration file, without running nginx nor the Lua code. 
```